docs/using-radicle/adding-a-seed-node.md

```diff
@@ -33,7 +33,7 @@ custom seed address, you can do so by following these steps:
    sidebar.
 
 Recent seed node activity and connected peers can be viewed on the added seed
-node's dashboard (e.g. http://seed.my.org). The default seed dashboard for
+node's dashboard (e.g. ht<span>tp:</span>//seed.my.org). The default seed dashboard for
```

Comment:
🤔

Reply:
It's to escape the links and not have them be codeblocks.

Reply:
It's genius, trust me

Reply:
http://seed.my.org

Reply:
I think a `codeblock` would be fine. Otherwise the next dev stumbling onto this will revert it, not knowing why it's like that.

Reply:
Ok makes sense 🥲